revision.go

```diff
@@ -0,0 +1,240 @@
+package driver
+
+import (
+	"context"
+	"github.com/arangodb/go-velocypack"
+	"path"
+)
+
+// RevisionInt64 is representation of '_rev' string value as an int64 number
+type RevisionInt64 int64
```

Comment:
I should have mentioned this earlier, sorry. On the `arangod` side we work with `uint64_t` instead of `int64_t`. I wonder whether we should use `uint64` here throughout.

Reply:
changed